source/includes/gcp/_externalips.md

```diff
@@ -133,4 +134,57 @@ Attributes | &nbsp;
 `kind`<br/>*string* | Type of the resource. Always compute#address for addresses.
 `shortUsers`<br/>*Array[string]* | The names of the instances the IP address is attached to.
 `shortRegion`<br/>*string* | A short version of the region name
-`type`<br/>*string* | One of EPHEMERAL or STATIC. EPHEMERAL are linked to an instance and are released automatically when an instance is deleted.
\ No newline at end of file
+`type`<br/>*string* | One of EPHEMERAL or STATIC. EPHEMERAL are linked to an instance and are released automatically when an instance is deleted.
+
+<!-------------------- RESERVE AN EXTERNAL IP -------------------->
+
+#### Reserve an external IP
+
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/externalips?operation=reserve"
+
+# Request example:
+```
+```json
+{
+  "name": "my-static-ip",
+  "shortRegion": "northamerica-northeast1",
+  "instanceId": "5611478403377505138"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/externalips</code>
+
+Reserve a new external static IP in a given [environment](#administration-environments).
+If instanceId is provided the IP will be reserved and attached to this instance.
```

Comment:
maybe put `instanceId` in a code block with back-ticks

Reply:
Done